test/test.js

```diff
@@ -37,12 +37,17 @@ function getTestOptions (settings) {
     return defaults({}, settings && settings.set, disabled);
 }
 
+function writeConfig (from, to) {
+    writeFile(`${to}/.git/config`, readFile(`${to}/.git/config`) + readFile(`${from}/gitconfig`));
```

Comment:
OK, it appears it doesn't work for me. But seems like it works if you run the following in testing repo dir:
```js
function colorGitOutput () {
    var gitColorCommands = [
        'git config color.branch.current red',
        'git config color.branch.local red',
        'git config color.branch.remote red',
        'git config color.diff.meta red',
        'git config color.diff.frag red',
        'git config color.diff.old red',
        'git config color.diff.new red',
        'git config color.status.added red',
        'git config color.status.changed red',
        'git config color.status.untracked red'
    ];

    return Promise.all(gitColorCommands.map(s => exec(s)));
}
```